src/Extensions/DisplayLogic.php

```diff
@@ -46,101 +43,102 @@ public function displayIf($master)
     }
 
     /**
      * If the criteria evaluate true, the field should hide.
      * The field will be hidden with CSS on page load, before the script loads.
-     * @param  string $master The name of the master field
-     * @return Criteria
+     *
+     * @param string $dispatcher The name of the form field that is controlling the display
      */
-    public function hideIf($master)
+    public function hideIf(string $dispatcher): Criteria
     {
         $class = "display-logic display-logic-hide";
         $this->owner->addExtraClass($class);
 
         if ($this->owner->hasMethod('addHolderClass')) {
             $this->owner->addHolderClass($class);
         }
-        return $this->setDisplayLogicCriteria(Criteria::create($this->owner, $master));
+        return $this->setDisplayLogicCriteria(Criteria::create($this->owner, $dispatcher));
     }
 
     /**
      * If the criteria evaluate true, the field should hide.
-     * The field will displayed before the script loads.
-     * @param  string $master The name of the master field
-     * @return Criteria
+     * The field will be displayed before the script loads.
+     *
+     * @param string $dispatcher The name of the form field that is controlling the display
      */
-    public function displayUnless($master)
+    public function displayUnless(string $dispatcher): Criteria
     {
-        return $this->owner->hideIf($master);
+        return $this->owner->hideIf($dispatcher);
     }
 
     /**
      * If the criteria evaluate true, the field should display.
      * The field will be hidden with CSS on page load, before the script loads.
-     * @param  string $master The name of the master field
-     * @return Criteria
+     *
+     * @param string $dispatcher The name of the form field that is controlling the display
      */
-    public function hideUnless($master)
+    public function hideUnless(string $dispatcher): Criteria
     {
-        return $this->owner->displayIf($master);
+        return $this->owner->displayIf($dispatcher);
     }
 
 
-    public function getDisplayLogicCriteria()
+    public function getDisplayLogicCriteria(): ?Criteria
     {
         $id = $this->owner->getName();
-        return isset($this->displayLogicCriteria[$id]) ? $this->displayLogicCriteria[$id] : null;
+
+        return $this->displayLogicCriteria[$id] ?? null;
     }
 
     /**
-     * A comma-separated list of the master form fields that control the display of this field
-     *
-     * @return  string
+     * A comma-separated list of the dispatcher form fields that control the display of this field
      */
-    public function DisplayLogicMasters()
+    public function DisplayLogicDispatchers(): ?string
     {
         if ($criteria = $this->getDisplayLogicCriteria()) {
-            return implode(",", array_unique($criteria->getMasterList()));
+            return implode(",", array_unique($criteria->getPrimaryList()));
```

Comment:
There's no other occurrence of either of these methods in the PR, so it probably needs changing somewhere else, too?